ci/OWNERS

```diff
@@ -143,10 +143,13 @@ nixos/modules/installer/tools/nix-fallback-paths.nix  @NixOS/nix-team @raitobeza
 /nixos/tests/amazon-ssm-agent.nix                 @arianvp
 /nixos/modules/system/boot/grow-partition.nix     @arianvp
 
+# Monitoring
+/nixos/modules/services/monitoring/fluent-bit.nix @samrose @fpletz
+/nixos/tests/fluent-bit.nix                       @samrose @fpletz
```

Comment:
Unfortunately, I'm not using fluent-bit anymore. Please remove me because I can't really maintain it. AFAIK @samrose wasn't really active in the last few years maintaining this package so if he doesn't respond it's probably best to remove him too.

You can add yourself and I'll be happy to review your changes though. :smile: